tools/make-kani-release/src/main.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This file is a glorified shell script for constructing a Kani release bundle.
+//! We use Rust here just to aid in making the "script" more robust.
+//!
+//! Run with `cargo run -p make-kani-release -- <version>` and this will produce
+//! (e.g.) `kani-1.0-x86_64-unknown-linux-gnu.tar.gz`.
+
+use std::{ffi::OsString, path::Path, process::Command};
+
+use anyhow::{bail, Result};
+
+fn main() -> Result<()> {
+    let version_string = parse_args()?;
+    let kani_string = format!("kani-{}", version_string);
+    let bundle_name = format!("{}-{}.tar.gz", kani_string, env!("TARGET"));
+
+    if !Path::new("src/kani-compiler").exists() {
+        bail!("Run from project root directory. Couldn't find 'src/kani-compiler'.");
+    }
+
+    let dir = Path::new(&kani_string);
+    if dir.exists() {
```

Comment:
Maybe add an `--override` option to make testing easier.

Reply:
I decided against for now. I've used this sort of release script a lot in the past and testing (running into problems during bundling) is pretty rare actually.

Note that this happily overwrites the `tar.gz` file. It's just the directory that only gets left existing if the bundling process crashes for some reason. So if you're trying to create the bundle over and over again (e.g. repeatedly running `cargo run -p make-kani-release -- latest`, that works fine.